src/HTTPClient.ts

```diff
@@ -61,6 +63,8 @@ export class HTTPClient {
 	 * @returns Nothing
 	 */
 	private parseMiddleWareArray(middlewareArray: Middleware[]) {
+		// The calling function should validate if middleware is not undefined or not empty
```

Comment:
why not adding this in the documentation of the function to it's shown in the intellisense?

Reply:
Done